daemon/prune.go

```diff
@@ -223,14 +223,10 @@ func (daemon *Daemon) NetworksPrune(pruneFilters filters.Args) (*types.NetworksP
 	clusterRep, err := daemon.clusterNetworksPrune(pruneFilters)
 	if err != nil {
 		logrus.Warnf("could not remove cluster networks: %v", err)
```

Comment:
Wondering if we should actually return this error instead; `clusterNetworksPrune()` only returns an error if `cluster.GetNetworks()` fails, which may be worth reporting instead of silently ignoring / only logging as warning